packages/core/stories/elements/Link/Link.stories.mdx

```diff
@@ -61,34 +63,37 @@ Links are used to embed actions or pathways to more information in a sentence or
   <Story story={stories.linkCalloutCSS} />
 </Canvas>
 
+---
 
-#### Link styled as a button
-**Usage:** Use when you are linking just a few words of text or when you are doing inline links. In general, we should style buttons as button and style links as links, to provide . [*For material honesty, we shouldn't encourage styling links as buttons, should use callout links instead, unless we can come up with appropriate examples*]
+### Link styled as a button
 
-**Style:**
+- **Usage:** If absolutely necessary this can be used for calls-to-action if there are technical reasons not to use `ma__button`.
```

Comment:
So "Link as a button" is using the `ma__button` className (expand the to show the HTML snippet), the difference here is that it's actually an anchor link `<a>` not a `<button>`.  Although there are use cases of in mayflower implementations, my opinion is that we should try to differentiate the styling between buttons and links, as they function differently, and that will help set expectations of the perceived behaviors.

Reply:
That makes sense